src/fin_webapp/app/services/alpha_vantage_service.rb

```diff
@@ -0,0 +1,32 @@
+# app/services/alpha_vantage_service.rb
+
+require 'faraday'
+require 'json'
+
+class AlphaVantageService
+  def initialize
+    @api_key = ENV['ALPHA_VANTAGE_API_KEY']
+    @connection = Faraday.new(url: 'https://www.alphavantage.co')

```

Comment:
convert the URL to a constant and store it

Reply:
63cae27 resolved